pocs/state/states/default/observing.py

```diff
@@ -1,7 +1,11 @@
+import time
+
 from pocs.utils import error
-from time import sleep
 
-wait_interval = 15.
+min_sleeping_interval = 0.1
+waiting_msg_interval = 15
```

Comment:
I'd worry about adding yet more output, especially since we don't really expect there to be any delay other than the intended 120 seconds (or whatever). I think that the message is mostly to satisfy us that things are happening while debugging.

Reply:
I changed the waiting time to 2.5 seconds but only did output every 30 seconds IIRC. Unfortunately that's on PAN001 and I still don't have access so can't see change.